redbadger/website-honestly
# People page layout modifications

### Motivation

- Update layout on team page and badger profile page
- Colours should be red instead of green
- The name and position should go below picture on team page
- Placeholder is now solid light grey

(Observation: because of branching issues, I had to rebase and remake this PR, which ended up on @dstodolny commits on the secondary image being carried along because it was merged to the old branch. This might result in circle CI not being able to build because it requires updates from badger brain)

### Test plan

- Check design on zeplin
- Check responsiveness on different browsers

### Pre-merge checklist
